atf_metrics/src/atf_metrics/error.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+class ATFAnalyserError(Exception):
```

Comment:
@floweisshardt rename to `ATFMetricsError` and `ATFMetricsConfigurationError`

Reply:
I don't know whether `ATFMetricsError` would express the correct meaning if it was `ATFAnalyserError` before...
you will have to check where that error is used and then rename/move/resolve accordingly

Reply:
agreed. let's keep `ATFAnalyserError`